<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SerialPort</class>
 <widget class="QWidget" name="SerialPort">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1242</width>
    <height>613</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:white;</string>
  </property>
  <widget class="QWidget" name="centralWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>800</width>
     <height>501</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>WenQuanYi Zen Hei</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <widget class="QPlainTextEdit" name="DataReceived">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>20</y>
      <width>360</width>
      <height>471</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>2</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>WenQuanYi Zen Hei Sharp</family>
      <pointsize>11</pointsize>
      <weight>7</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 57 11pt &quot;WenQuanYi Zen Hei Sharp&quot;;</string>
    </property>
    <property name="text">
     <string>数据收发日志</string>
    </property>
   </widget>
   <widget class="QFrame" name="Leftwindow">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>360</width>
      <height>470</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(30, 14, 255);

background-color: rgb(220, 233, 255);
</string>
    </property>
    <widget class="QLabel" name="StationCode">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(24, 27, 255);</string>
     </property>
     <property name="text">
      <string>设备编号</string>
     </property>
    </widget>
    <widget class="QLabel" name="TNS">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>20</y>
       <width>110</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>WenQuanYi Zen Hei Sharp</family>
       <pointsize>12</pointsize>
       <weight>7</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: black;
font: 57 12pt &quot;WenQuanYi Zen Hei Sharp&quot;;
border : 0.5px solid  grey;
border-radius:5px;
background-color:white;

</string>
     </property>
     <property name="midLineWidth">
      <number>1</number>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="n_line1">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>100</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(24, 27, 255);
</string>
     </property>
     <property name="text">
      <string>雨      量</string>
     </property>
    </widget>
    <widget class="QLabel" name="Rainfall">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>100</y>
       <width>110</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>WenQuanYi Zen Hei Sharp</family>
       <pointsize>13</pointsize>
       <weight>7</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: black;
font: 57 13pt &quot;WenQuanYi Zen Hei Sharp&quot;;
border : 0.5px solid  grey;
border-radius:5px;
background-color:white;</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="d_line1">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>100</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>WenQuanYi Zen Hei Sharp</family>
       <pointsize>14</pointsize>
       <weight>7</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(24, 27, 255);
font: 57 14pt &quot;WenQuanYi Zen Hei Sharp&quot;;</string>
     </property>
     <property name="text">
      <string>mm</string>
     </property>
    </widget>
    <widget class="QLabel" name="Messagetime">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>100</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(200, 7, 17);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="n_line2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>180</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(24, 27, 255);</string>
     </property>
     <property name="text">
      <string>水      位</string>
     </property>
    </widget>
    <widget class="QLabel" name="Waterlevel">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>180</y>
       <width>110</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>WenQuanYi Zen Hei Sharp</family>
       <pointsize>13</pointsize>
       <weight>7</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: black;
font: 57 13pt &quot;WenQuanYi Zen Hei Sharp&quot;;
border : 0.5px solid  grey;
border-radius:5px;
background-color:white;</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="d_line2">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>180</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>WenQuanYi Zen Hei Sharp</family>
       <pointsize>14</pointsize>
       <weight>7</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(24, 27, 255);
font: 57 14pt &quot;WenQuanYi Zen Hei Sharp&quot;;</string>
     </property>
     <property name="text">
      <string>m</string>
     </property>
    </widget>
    <widget class="QLabel" name="n_line3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>260</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(24, 27, 255);</string>
     </property>
     <property name="text">
      <string>流      速</string>
     </property>
    </widget>
    <widget class="QLabel" name="Flowvelocity">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>260</y>
       <width>110</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>WenQuanYi Zen Hei Sharp</family>
       <pointsize>13</pointsize>
       <weight>7</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: black;
font: 57 13pt &quot;WenQuanYi Zen Hei Sharp&quot;;
border : 0.5px solid  grey;
border-radius:5px;
background-color:white;</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="d_line3">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>260</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>WenQuanYi Zen Hei Sharp</family>
       <pointsize>14</pointsize>
       <weight>7</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(24, 27, 255);
font: 57 14pt &quot;WenQuanYi Zen Hei Sharp&quot;;</string>
     </property>
     <property name="text">
      <string>m/s</string>
     </property>
    </widget>
    <widget class="QLabel" name="n_line4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>340</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(24, 27, 255);
</string>
     </property>
     <property name="text">
      <string>流      量</string>
     </property>
    </widget>
    <widget class="QLabel" name="Flow">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>340</y>
       <width>110</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>WenQuanYi Zen Hei Sharp</family>
       <pointsize>13</pointsize>
       <weight>7</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: black;
font: 57 13pt &quot;WenQuanYi Zen Hei Sharp&quot;;
border : 0.5px solid  grey;
border-radius:5px;
background-color:white;</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="d_line4">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>340</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>WenQuanYi Zen Hei Sharp</family>
       <pointsize>14</pointsize>
       <weight>7</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(24, 27, 255);
font: 57 14pt &quot;WenQuanYi Zen Hei Sharp&quot;;</string>
     </property>
     <property name="text">
      <string>m³/s</string>
     </property>
    </widget>
    <widget class="QLabel" name="n_line5">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>420</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(24, 27, 255);
</string>
     </property>
     <property name="text">
      <string>累计流量</string>
     </property>
    </widget>
    <widget class="QLabel" name="Totalflow">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>420</y>
       <width>110</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>WenQuanYi Zen Hei Sharp</family>
       <pointsize>13</pointsize>
       <weight>7</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: black;
font: 57 13pt &quot;WenQuanYi Zen Hei Sharp&quot;;
border : 0.5px solid  grey;
border-radius:5px;
background-color:white;</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="d_line5">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>420</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>WenQuanYi Zen Hei Sharp</family>
       <pointsize>14</pointsize>
       <weight>7</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(24, 27, 255);
font: 57 14pt &quot;WenQuanYi Zen Hei Sharp&quot;;</string>
     </property>
     <property name="text">
      <string>m³</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="Topwindow" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>WenQuanYi Zen Hei</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(85, 170, 127);</string>
   </property>
   <widget class="QLabel" name="ruanjian">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>200</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
</string>
    </property>
    <property name="text">
     <string>视频遥测终端管理软件</string>
    </property>
   </widget>
   <widget class="QLabel" name="Timeshow">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>30</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>WenQuanYi Zen Hei Sharp</family>
      <pointsize>12</pointsize>
      <weight>7</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
font: 57 12pt &quot;WenQuanYi Zen Hei Sharp&quot;;

background-color: rgb(85, 170, 127);</string>
    </property>
    <property name="text">
     <string>time</string>
    </property>
   </widget>
   <widget class="QPushButton" name="serial_btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="Bottomwidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>550</y>
     <width>800</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
</string>
   </property>
   <widget class="QPushButton" name="setting_btn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Batteryvoltage">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>16</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>WenQuanYi Zen Hei Sharp</family>
      <pointsize>14</pointsize>
      <weight>7</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(94, 145, 255);
background-color:transparent;
font: 57 14pt &quot;WenQuanYi Zen Hei Sharp&quot;;</string>
    </property>
    <property name="text">
     <string>13.2</string>
    </property>
   </widget>
   <widget class="QLabel" name="V">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>16</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>WenQuanYi Zen Hei Sharp</family>
      <pointsize>14</pointsize>
      <weight>7</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(94, 145, 255);
background-color:transparent;
font: 57 14pt &quot;WenQuanYi Zen Hei Sharp&quot;;</string>
    </property>
    <property name="text">
     <string>V</string>
    </property>
   </widget>
   <widget class="QLabel" name="SWJTU">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>16</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="battery_btn">
    <property name="geometry">
     <rect>
      <x>307</x>
      <y>10</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="G4_btn">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>40</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Eth1_btn">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Eth2_btn">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Poweroff_btn">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>10</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="signal_strength">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>16</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>WenQuanYi Zen Hei Sharp</family>
      <pointsize>14</pointsize>
      <weight>7</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(94, 145, 255);
background-color:transparent;
font: 57 14pt &quot;WenQuanYi Zen Hei Sharp&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>Poweroff_btn</sender>
   <signal>clicked()</signal>
   <receiver>centralWidget</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>105</x>
     <y>575</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>300</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
